MUSEOMICS CLARIFIES INTRASPECIFIC STRUCTURE IN THE TRITON COCKATOO (CACATUA TRITON)

Arthur F. Sands 1 2, Astrid A.L. Andersson 1, Geraldine Lee 2, Taylor Hains 3 4, Kerry Reid 1 5, Leo Joseph 6, Caroline Dingle 1 7, Frank E. Rheindt 2, Juha Meril√§ 1 8

1 Area of Ecology & Biodiversity, School of Biological Sciences, The University of Hong Kong, Hong Kong
2 Department of Biological Sciences, National University of Singapore, Singapore 
3 Committee on Evolutionary Biology, University of Chicago, Chicago, USA 
4 Negaunee Integrative Research Center, Field Museum of Natural History, Chicago, USA 
5 
6 Australian National Wildlife Collection, CSIRO National Research Collections Australia, Canberra, Australia
7 Biology Department, Capilano University, North Vancouver, Canada
8 Ecological Genetics Research Unit, Faculty of Biological and Environmental Sciences, University of Helsinki, Helsinki, Finland

Corresponding author: sands@hku.hk

This repository relates to scripts and support files of the above-mentioned paper. Queries related to scripts and provided files can be directed to the corresponding author.

######################## WGR-processing ######################## 
Processing batch scripts and support files examples for whole-genome resequencing data can be found in the "WGR-processing" directory. 
These scripts and support file examples have been labelled in order of processing from Step1-x. 
The scripts have been constructed generically where pathways, input file names and output names require specification. 
Notes have been given at the top of each script (where possible) to provide basic instructions. 
Given the initiation of different software/packages may be called differently on different high-performance computers, lines of code activating/running software/packages should be added by the user and are not specified.

######################## Analytics ############################ 
Processing batch and R scripts and supporting files for specific analysis can be found in the "Analytics" directory. 
These scripts and support files have been placed in subdirectories named according to the analysis and are labelled in order of processing from Step1-x. 
For the most part, the scripts are given with pathways, input file names and output names exactly how they were run on the high-performance computer Puhti (CSS) or on the authors' personal computers. 
Notes have been given at the top of each script (where possible) to provide basic instructions.

################## Data files & raw results ################### 
Basic starting VCF files and raw/preliminary results can be found on the Dryad Digital Repository (see )
